module/framework.js

```diff
@@ -431,7 +431,11 @@ export function parsedContentToHTML(content) {
 
     // Set image source
     let img = new Image();
-    img.src = `https://worldanvil.com${i.dataset.src}`;
+    if (i.dataset.src.startsWith("/")) {
+      img.src = `https://worldanvil.com${i.dataset.src}`;
+    } else {
+      img.src = i.dataset.src;
+    }
```

Comment:
Code style to condense slightly:
```suggestion
    img.src = (i.dataset.src.startsWith("/") ? "https://worldanvil.com" : "") + i.dataset.src;
```